Lib/test/test_threading.py

```diff
@@ -123,6 +123,31 @@ def func(): pass
             thread = threading.Thread(target=func)
             self.assertEqual(thread.name, "Thread-5 (func)")
 
+    def test_args_argument(self):
+        # bpo-45735: Using list or tuple as *args* in constructor could
+        # achieve the same effect.
+        num_list = [1]
+        num_tuple = (1,)
+
+        str_list = ["str"]
+        str_tuple = ("str",)
+
+        list_in_tuple = ([1],)
+        tuple_in_list = [(1,)]
+
+        test_cases = [
+            [num_list, lambda arg: self.assertEqual(arg, 1)],
+            [num_tuple, lambda arg: self.assertEqual(arg, 1)],
+            [str_list, lambda arg: self.assertEqual(arg, "str")],
+            [str_tuple, lambda arg: self.assertEqual(arg, "str")],
+            [list_in_tuple, lambda arg: self.assertEqual(arg, [1])],
+            [tuple_in_list, lambda arg: self.assertEqual(arg, (1,))]
+        ]
+        for test_case in test_cases:
```

Comment:
Very minor: the test cases should be tuple (they are heterogenous pairs, not lists that could be sorted/expanded/etc), and `for (args, target) in test_cases` could be a more readable loop.

Reply:
Thanks for your suggestions! It is very helpful for me. :-)